docs/Data correction documentary-cleared.ipynb

```diff
@@ -0,0 +1,301 @@
+    "- Note: until now, we have treated $T^{g, p|a}$ as constant parameter and ignored its wavelength-dependence, as its variation with wavelength is small enough to not impact the correction procedure. However, to keep it generic and enable to insert a wavelength-dependent $T^{g, p|a}(\\lambda)$, the dependence on wavelength will be pre-characterized (during fabrication of the cell) and inserted as function into this workflow instead of a scalar.\n",
+    "\n",
+    "$\\mathbf{Workflow 2}$:<br>\n",
+    "A more precise data correction is based on a measure of $p$ by a direct-beam measurement through the depolarized cell (i.e., $P_{3He}(t) \\rightarrow 0$)(Ref. 2), leading to \"workflow 2\". For $P_{3He}(t) \\rightarrow 0$ and an unpolarized incident beam on the depolarized cell(i.e., $\\hat{T}_{depol} = (\\hat{T}^++\\hat{T}^-)/2$), the neutron transmission \\eqref{eq:T} becomes:\n",
+    "\n",
+    "$$T^{depol, p|a}(\\lambda)=T^{g, p|a}(\\lambda)e^{-O^{0, p|a}*\\lambda}$$\n",
+    "\n",
+    "This leads to following derivation by neutron experiment:\n",
+    "\n",
+    "$$I^{DB,depol,p|a}_{\\lambda}/I^{DB,no-cell}_{\\lambda}=T^{g, p|a}(\\lambda)e^{-O^{0, p|a}*\\lambda}$$\n",
+    "\n",
+    "For a measurement of $I^{DB,depol,p|a}_{\\lambda}$, the $^3$He cell needs to be depolarized at the end of its lifetime and a direct-beam measurement through each cell (and the other cell out of beam) has to be performed without sample. \n",
+    "\n",
+    "- Note: Workflow 2 should overwrite workflow 1 opacity after the experiment\n",
+    "- Note: Different to the sample data, $I^{DB,depol,p|a}$ and $I^{DB,no-cell}$ are not time-dependent\n",
+    "- Note: It is important to fit $I_{depol}/I_0$ instead of $I_{depol}$, such that all the parameters in the fit-functions are scalars.\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "14be6c8e-3ee3-4ffc-a8f0-e9face4c4d27",
+   "metadata": {},
+   "source": [
+    "#### 4.2 Calculation of time-dependent 3He-polarization\n",
+    "  \n",
+    "Now $O^{p|a}(\\lambda)=)O^{0, p|a}*\\lambda$ is known. <br>\n",
+    "The next step for getting the transmission functions $\\hat{T}^{\\pm}$ is to calculate the nuclear polarization of the $^3$He-cell, which decays exponentially with a decay-time $T^{1, p|a}$ and is given by:\n",
+    "\n",
+    "$$P^{3He, p|a}(t) = C^{p|a}e^{-t/T^{1, p|a}}$$.\n",
+    "\n",
+    "From the neutron measurement, this again can be retrieved by measuring the transmission through each cell (i.e., for an unpolarized incoming beam into the polarized cell):\n",
+    "\n",
+    "$$\\hat{T}^{unpol-in}(\\lambda, t) = T^{g, p|a}(\\lambda)e^{-O_0^{p|a}*\\lambda}\\cosh({P_{3He}(t)*O^{0, p|a}*\\lambda})$$<br>\n",
+    "$$I^{DB,unpol-in,p|a}_{\\lambda, t}/I^{DB, no-cell}_{\\lambda} = T^{g, p|a}(\\lambda)e^{-O^{0, p|a}*\\lambda}\\cosh({P^{3He, p|a}(t)*O^{0, p|a}*\\lambda})$$\n",
```

Comment:
Use `\frac` for more readable fractions?

Reply:
agreed